python/nano/src/bigdl/nano/pytorch/inference/optimizer.py

```diff
@@ -780,6 +810,7 @@ def trace(model: nn.Module,
             use_jit = (accelerator == "jit")
             return PytorchIPEXJITModel(model, input_sample=input_sample, use_ipex=use_ipex,
                                        use_jit=use_jit, channels_last=channels_last,
+                                       channels_last_available=channels_last_available,
                                        thread_num=thread_num, inplace=inplace,
                                        jit_strict=jit_strict)
         invalidInputError(False, "Accelerator {} is invalid.".format(accelerator))
```

Comment:
we want some unit test for this fix in https://github.com/intel-analytics/BigDL/blob/main/python/nano/test/pytorch/tests/inference/test_ipex_jit_inference.py

Reply:
we need to make sure `channels_last_available` is well recovered even after `InferenceOptimizer.save/load`